src/gentropy/dataset/l2g_feature_matrix.py

```diff
@@ -36,7 +36,7 @@ def __init__(
             with_gold_standard (bool): Whether to include the gold standard set in the feature matrix.
         """
         self.with_gold_standard = with_gold_standard
-        self.fixed_cols = ["studyLocusId", "geneId"]
+        self.fixed_cols = ["studyLocusId", "geneId", "traitFromSourceMappedId"]
```

Comment:
Feature matrix doesn't have a schema.
But I need to define that this column is expected as one of the fixed columns to pivot on.